Backend/app/rider-platform/rider-app/Main/src/Domain/Action/Dashboard/IssueList.hs

```diff
@@ -47,6 +55,12 @@ getIssueList merchantShortId mbLimit mbOffset mbmobileCountryCode mbMobileNumber
       mobileNumberDbHash <- getDbHash mobileNumber
       let mobileCountryCode = fromMaybe mobileIndianCode mbmobileCountryCode
       customer <- runInReplica $ QPerson.findByMobileNumberAndMerchantId mobileCountryCode mobileNumberDbHash merchant.id >>= fromMaybeM (PersonNotFound mobileNumber)
-      runInReplica $ QIssue.findByCustomerId customer.id mbLimit mbOffset fromDate toDate
+      issueList <- runInReplica $ QIssue.findByCustomerId customer.id mbLimit mbOffset fromDate toDate
+      let count = length issueList
+      let summary = Summary {totalCount = count, count}
```

Comment:
Why we need both `totalCount` and `count` and can't have only one of those?

Reply:
yeah it is the required format for UI to work